loops-and-arrays.js

```diff
@@ -41,6 +54,12 @@ const numbers = [2, 4, 5, 29, 38];
 // ==========================================
 
 const squares = [30, 2, 8, 24, 11];
+console.log();
+console.log(squares);
+for (v = 0; v < squares.length; v++){
+    result = squares[v] * squares[v] * squares[v];
```

Comment:
het is beter om van result een variabe;e te maken met const ervoor 

dit is de enige loop waarbij je de originele array niet overschreven hebt. Dat kan er nog aan toegevoegd worden. 
Vervolgens kan de squares array buiten de for loop worden gelogd in de terminal